src/support/string-utils.ts

```diff
@@ -0,0 +1,3 @@
+export const preserveSurroundingWhitespace =
+    (original: string, target: string): string => original.match(/^\s*/)![0] +
+    target.replace(/^\s*|\s*$/, '') + original.match(/\s*$/)![0];
```

Comment:
Need to add `/g` to the replace regexp or else this only matches leading whitespace.

Reply:
I AM SO EMBARASSED.

Fixed!